LocalDynamoDb.Tests/IntegrationTest.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Amazon.DynamoDBv2;
+using Amazon.DynamoDBv2.Model;
+using Xunit;
+
+namespace LocalDynamoDb.Tests
+{
+    public static class IntegrationTest
+    {
+        [Fact]
+        public static async Task FullIntegrationTest()
+        {
+            var localDynamo = new LocalDynamo(8001);
+
+            try
+            {
+                localDynamo.Start();
+                await CreateTestTable(localDynamo.Client);
+
+                var tables = await localDynamo.Client.ListTablesAsync();
+                Assert.True(tables.TableNames.Count == 1, "Wrong number of tables.  Expected 1 but got " + tables.TableNames.Count);
+            }
+            finally
+            {
+                localDynamo.Stop();
+            }
+        }
+
+        private static async Task CreateTestTable(IAmazonDynamoDB dynamoClient)
+        {
+            Console.WriteLine("Creating tables.");
```

Comment:
Should use `ITestOutputHelper` as otherwise you might not ever see this in xunit as it re-routes the console.